cstar/roms/simulation.py

```diff
@@ -1408,14 +1389,19 @@ def run(
                 + "\n If you have already run Simulation.build(), either run it again or "
                 + " add the executable path manually using Simulation.exe_path='YOUR/PATH'."
             )
+
         if self.discretization.n_procs_tot is None:
             raise ValueError(
                 "Unable to calculate node distribution for this Simulation. "
                 + "Simulation.n_procs_tot is not set"
```

Comment:
unnecessary string concatenations should be replaced with implicit concatenations (no need for `+`)